### DISCUSSION TITLE
Option to download APK?

### DISCUSSION THREAD

**Discussion by whee1736** [uxbox/uxbox#522] [General]
Is there currently an option to download an APK for testing?

**Comment by Alotor** [comment]
Hi, I think there is a misunderstanding. The application is web, so you can try it out at https://design.penpot.app/

No APK's needed ;)

Best,

**Comment by whee1736** [comment]
I should probably clarify. I didn't mean an APK for the Penpot application.

Is there an option to download our projects as an APK, or are the projects strictly for web based applications?

**Reply by Manueljlin** [reply]
It's an user interface and user experience design tool of the likes of Adobe XD, Figma, Gravit Designer, Sketch, Icons8's Lunacy and others - that means that it is exclusively for making scalable vector graphics files which are created to represent the intended look and user experience of a certain project plus some built in prototyping features. This tool isn't intended for creating an application or a website's code but rather as a way to design it and hand off to a developer. For what I understand on this reply, you are looking for something like Webflow?

**Reply by Alotor** [reply]
@Manueljlin is right. But one of the things that it's high in the backlog is to make the "viewer" side working on mobile so you can look at it on your mobile phone and check if the design is alright.

**Reply by whee1736** [reply]
@Alotor Yes! Perfect. I'm glad to hear a feature like this is coming in the future.
